src/Props/TeamCardProps/cards.ts

```diff
@@ -0,0 +1,139 @@
+import { CardProps } from "Props/props"
```

Comment:
I think a lot of the changes aren't supposed to be here. You might want to delete these and merge from main.